Tests/scripts/infrastructure_tests/validate_files_test.py

```diff
@@ -0,0 +1,38 @@
+from Tests.scripts.validate_files import FilesValidator
+# from Tests.scripts.hook_validations.conf_json import ConfJsonValidator
+
+
+def packagify_mock_no_change(modified, added, removed, tag):
+    return modified, added, removed
+
+
+def test_get_modified_files(mocker):
+    mocker.patch('Tests.scripts.hook_validations.conf_json.ConfJsonValidator.load_conf_file', return_value={})
+    file_validator = FilesValidator()
+
+    changed_files = '''A       Integrations/Recorded_Future/CHANGELOG.md
```

Comment:
add test to check changed yml
add test to packagify

Reply:
done